File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/swift/ADV-20.swift
import Foundation

func f(_ n: Int64) -> Int64 {
    if n == 1000000 { return 0 }
    return 1 + f(n + 1)
}

print("ADV-START"); fflush(stdout)
let n = Int64(readLine()!)!
let r = f(n)
print("OBS=R:\(r)"); fflush(stdout)
print("ADV-END"); fflush(stdout)

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/kotlin/ADV-20.kt
fun f(n: Long): Long {
    if (n == 1000000L) {
        return 0
    }
    return 1 + f(n + 1)
}

fun main() {
    println("ADV-START")
    System.out.flush()
    val start: Long = readln().toLong()
    val r = f(start)
    println("OBS=R:" + r)
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}
